spec/features/versions_spec.rb

```diff
@@ -218,6 +218,15 @@
   scenario 'display changes in splashpages', feature: true, versioning: true, js: true do
     visit admin_conference_splashpage_path(conference.short_title)
     click_link 'Create Splashpage'
+    uncheck('Display tracks on the splashpage?')
```

Comment:
I think it would look better if we leave the 'on the splashpage?' bit. It seems too repetitive. @Ana06 What do you think?